3-platform/shopping-cart/app/src/main/java/com/_6core/platform/shopping/cart/app/validator/Validator.java

```diff
@@ -0,0 +1,5 @@
+package com._6core.platform.shopping.cart.app.validator;
+
+public interface Validator<T> {
```

Comment:
I think we need to restrict type
```suggestion
public interface Validator<T extend ShoppingCartV01> {
```